pyemu/en.py

```diff
@@ -480,7 +480,7 @@ def _gaussian_draw(
                     # cnames = names
                     cnames = []
                     snames = set(names)
-                    for n in cov.names:
+                    for n in cov.col_names:
```

Comment:
I think `.names` should work so long as the `cov` object that is passed to the method is `Cov` type not `Mat`.  Does this not work for you?

Reply:
I never would have seen that line if I hadn't gotten an error there, but that doesn't mean I didn't screw something up. Looking around I see "cov.row_names" so I tried "cov.col_names" and voila, it worked.

Reply:
as `Cov` is the expected form `col_names` should be the same as `row_names` and both should be covered by that `names` property. So `col_names` should work but `names` is a handy implicit test for that `Cov` variable actually being a `Cov` object.